src/components/CommentForm/CommentForm.tsx

```diff
@@ -359,13 +360,7 @@ export const CommentForm = ({
 
     return (
         <>
-            <form
-                className={formWrapper}
-                onSubmit={e => {
-                    e.preventDefault();
-                    submitForm();
-                }}
-            >
+            <form className={formWrapper}>
```

Comment:
why was `onSubmit` logic removed?

Reply:
Because we can't pass `type='submit'` to `Button`. Unless I'm wrong about that?

Reply:
You can:
```
const Button = ({
	priority,
	size,
	icon: iconSvg,
	iconSide,
	cssOverrides,
	children,
	...props
}: ButtonProps) =>
```
it will be extracted and applied using `...props`

Reply:
Done